libs/wingc/src/ast.rs

```diff
@@ -309,6 +309,7 @@ pub enum UtilityFunctions {
 	Log,
 	Assert,
 	UnsafeCast,
+	Nodeof,
```

Comment:
Can you please add `NodeOf` to the language reference doc?

Reply:
Done